webpack.config.js

```diff
@@ -1,12 +1,19 @@
- const path = require('path');
+/* eslint-disable */
```

Comment:
- [OPTIONAL] it is always a good practice to make sure you fix all linter errors instead of disabling them, A linter is great for identifying errors when you use standard rules. Remember, a linter analyzes your code for stylistic and programming errors against the rules it knows. 💯 😉